README.md

```diff
@@ -24,12 +24,14 @@ an issue or pull-request adding assets.
 - BSV (Bitcoin Satoshi Vision)
 - BCH (Bitcoin Cash)
 - XVG (Verge)
-- USDC (USD Coin)
 - USDT (USD Tether)
 - XRP (Ripple)
-- TRX (Tron)
 - ARB (Arbitrum)
 - BSC (Binance Smart Chain)
+- USDC (USD Coin)
+- TRX (Tron)
+- SOL (Solana)
+- BNB (Binance Coin) - removed from the SDN as of 2025-11-24
```

Comment:
Has this been just combined with ETH addresses? Many of the EVM addresses are shared across chains

Reply:
> Has this been just combined with ETH addresses? Many of the EVM addresses are shared across chains

Unknown. All that I did was to review every digital currency type presently in the SDN list and enumerate it iin the README, and the corresponding, but unused, array in the code. The array is unused because I asked Nick to modify the code to handle any digital currency type enumerated in the SDN list (rather than looking at the array).

We could remove the array without issue, I believe.

Parenthetically, I note that the SDN list now has its first SOL address...